src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py

```diff
@@ -180,7 +181,7 @@ def run_conversion(
             use_times=use_times,
             write_as=write_as,
             write_electrical_series=write_electrical_series,
-            es_key=es_key,
+            es_key=es_key or self.es_key,
```

Comment:
from https://github.com/catalystneuro/neuroconv/pull/218/files#r1031812255, of course they can always override it and that will take priority at this line